docs/products/clickhouse/howto/use-play.rst

```diff
@@ -0,0 +1,14 @@
+Use ClickHouse® play UI
```

Comment:
A suggestion to use Aiven for ClickHouse® throughout the document.

Reply:
The play UI is not something we built, but rather what is available with ClickHouse. So I intentionally didn't put "Aiven for ClickHouse® play UI" not to mislead that this is part of our product.

Reply:
Same for another entry of "ClickHouse"